feat: Start preparing the final wording of the app description for the Play Store and the App Store (#31)
openfoodfacts/fastlane-descriptions-smoothie: fastlane/android/metadata/en-US/short_description.txt

> A more personal, helpful & knowledgeable Open Food Facts, still private by default!

Author: ```suggestion
A more personal, helpful & knowledgeable Open Food Facts, private by default!
```

> Open Food Facts, reimagined. More personal, helpful & knowledgeable, and still private by default!

Author: ```suggestion
A more personal, helpful & knowledgeable Open Food Facts, still private by default!
```

> Open Food Facts, reimagined. More personal, More helpful & knowledgeable, and still private by default!

Author: ```suggestion
Open Food Facts, reimagined. More personal, helpful & knowledgeable, and still private by default!
```

> This is the new version of Open Food Facts. More personal, More helpful and knowledgeable, and still private by default.

Author: ```suggestion
Open Food Facts, reimagined. More personal, More helpful & knowledgeable, and still private by default!
```
